BUILD

```diff
@@ -703,6 +702,7 @@ cc_library(
     "include/grpc++/status.h",
     "include/grpc++/status_code_enum.h",
     "include/grpc++/stream.h",
+    "include/grpc++/thread_pool.h",
```

Comment:
Should we do the same with all the relevant tests as well like the thread pool test